src/experiment_handler.py

```diff
@@ -22,12 +22,13 @@
 import importlib
 import threading
 import pickle
-
 from utils.options import Options
 from utils import socket_operations
 from experiment_prototype.experiment_exception import ExperimentException
 from experiment_prototype.experiment_prototype import ExperimentPrototype
 
+from utils import log_config
+log = log_config.log()
```

Comment:
This is a temporary fix to allow this file to be exported and ran outside of main. We will want a better solution in the future to fix this problem for all our modules.